<launch>
  <node pkg="pi_face_tracker" name="face_tracker" type="face_tracker.py" output="screen">
    <rosparam>
      input_image: /camera/image_raw
      flip_image: False
      do_face_tracking: True
      detect_face_only: False
      use_depth_for_detection: False
      use_depth_for_tracking: False
      min_features: 50
      abs_min_features: 6
      show_text: True
      show_features: True	
    </rosparam>
    
  	<param name="cascade_frontal_alt" value="$(find pi_face_tracker)/params/haarcascade_frontalface_alt.xml" />
   	<param name="cascade_frontal_alt2" value="$(find pi_face_tracker)/params/haarcascade_frontalface_alt2.xml" />
  	<param name="cascade_profile" value="$(find pi_face_tracker)/params/haarcascade_profileface.xml" />

  </node>
</launch>
